docs/intro/install.rst

```diff
@@ -130,6 +130,8 @@ To install Scrapy on Windows using ``pip``:
 
 Now, you should be able to :ref:`install Scrapy <intro-install-scrapy>` using ``pip``.
 
+To install Scrapy on Windows with `WSL`_, follow the Unbuntu instructions.
```

Comment:
- `` `WSL`_ `` is a broken reference.
- Unbuntu → Ubuntu.
- Can you link “Ubuntu instructions” to the section, for a slightly better user experience?